script.js

```diff
@@ -1,19 +1,19 @@
-const display = document.querySelector("#display-data-done-main");
+const display = document.querySelector(".display-data-done-main");
 
 /* FETCH JSON DATA */
 
 
 const getData = () =>{
-  fetch('./user.json')
+  return fetch('./user.json')
   .then(res => res.json())
   .then(data=>{;
     console.log(data);
    return data;
   })
 }
 
-const displayData = () =>{
-  const payload = getData();
+const displayData = async () =>{
+  const payload = await getData();
```

Comment:
Then we can `await` the promise to resolve. In order to use `await`, we have to make this function asynchronous by adding `async` before the arguments. using `async`/`await` is kinda 'newish'. Previously, you would run the promise and then use `.then` afterwards like you do above in the `getData` function.